### DISCUSSION TITLE
Weird problems importing an app into self-hosted system

### DISCUSSION THREAD

**Discussion by kpturner** [Budibase/budibase#10342] [Help] [answered]
I exported an app from the cloud and got a `.tar.gz`

I started a self-hosted BB instance using docker-compose and went to http://localhost:10000

I had the choice of creating a super user or importing an app.  I tried the latter using the `tar.gz` but that just popped up an error saying "Import failed". No other clues.

So I then created a super user and logged on.  

I was then presented with no option other than to create my first app.  No option to import or anything - just a "Lets go!" button.

So I then created an app (I don't need) from the sample data and now I have the ellipsis menu at the top left where I can "Exit to portal".

From here I can click "Create new app" -> "Import app" and that works.

I then "Exit to portal" to be able to delete the app that I had to create in the first place to get to this point.

I get the feeling this is not the expected UX to import an app from scratch?   What did I do wrong?

**Comment by melohagan** [comment]
Hey @kpturner 

This doesn't same right: 
> I had the choice of creating a super user or importing an app.

You should not be able to import an app until a super user has been created. I'll try to reproduce this and create a bug issue if I find a problem.

**Comment by melohagan** [comment, marked as answer]
@kpturner Had a discussion with the team, and it appears that this is a legacy feature that we intend to remove.

The original purpose was to bulk import apps, which would also include the admin user. 

In future when setting up Budibase, simply create a new admin user and then import your app after you are setup. As you mentioned, you may need to create the *My first app* if it's the very first time setting up the Budibase instance, but it only takes a second and you can navigate back to the app portal to import your app.

**Comment by kpturner** [comment]
OK thanks
